packages/core/src/L3/nodejs/scripts/index.ts

```diff
@@ -0,0 +1 @@
+export { CleanScript } from "./CleanScript";
```

Comment:
Do we need to export `CleanScriptPath` too, or do we consider it to be an implementation detail?

Reply:
yes, forgot